GUI-pause-menu/Assets/UI/Uss/PauseMenu.uss

```diff
@@ -0,0 +1,74 @@
+.menu-button {
+    padding-top: 12px;
+    padding-bottom: 12px;
+    font-size: 31px;
+    border-color: rgb(101, 101, 101);
+    border-left-width: 0;
+    border-right-width: 1px;
+    border-top-width: 0;
+    border-bottom-width: 0;
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    color: rgba(238, 238, 238, 0.95);
+    margin-left: 0;
+    margin-right: 0;
+    margin-bottom: 0;
+    flex-grow: 1;
+    background-color: rgba(0, 0, 0, 0);
+    border-right-color: rgb(0, 0, 0);
+    -unity-text-align: middle-center;
+    -unity-font: url('/Assets/Fonts/Teko-Regular.ttf');
+}
+
+.menu {
+    flex-direction: row;
+    background-color: rgba(0, 0, 0, 0.3);
+    width: 100%;
+    padding-left: 0;
+    padding-right: 0;
+    padding-top: 0;
+    padding-bottom: 0;
+}
+
+.screen {
+    flex-grow: 1;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    background-color: rgba(231, 0, 0, 0.46);
+    flex-shrink: 0;
+}
+
+.menu-container {
+    flex-direction: row;
+    flex-grow: 0;
+    justify-content: center;
+    background-color: rgba(0, 0, 0, 0.76);
+    width: 600px;
+    align-items: center;
+    flex-shrink: 0;
+}
+
+.menu-button:hover {
+    border-color: rgb(255, 255, 255);
+    background-color: rgba(0, 0, 0, 0.82);
+    border-left-width: 0;
+    border-right-color: rgb(0, 0, 0);
+}
+
+.menu-button:active {
+    background-color: rgb(0, 0, 0);
+}
+
+.screen-placeholder {
+    margin-top: 40px;
+    margin-bottom: 40px;
+}
```

Comment:
margin: 40px, 0, 40px, 0

Reply:
Niet mogelijk USS, USS wordt automatisch genereert door Unity. Helaas is USS nog niet zo ver als CSS.